src/Controller/CRUDController.php

```diff
@@ -997,15 +997,18 @@ final public function setTwigGlobals(Request $request): void
         $twig = $this->container->get('twig');
         \assert($twig instanceof Environment);
 
-        $twig->addGlobal('admin', $this->admin);
+        try {
```

Comment:
Should we split this in two separates try/catch ?

Reply:
Done